Makefile

```
@@ -204,3 +204,18 @@ konduit-snapshot: get-cluster-credentials
 	} & \
 	bin/konduit.sh -d s189p01-cpdnpq-pd-pg-snapshot -k s189p01-cpdnpq-pd-app-kv npq-registration-production-web -- psql > "$$tmp_file"
 	exit 0
+
+# Creates a konduit to the migration DB and points development to
+# it. The konduit URL is removed when the konduit is closed.
+konduit-migration: get-cluster-credentials
+	trap 'make konduit-cleanup' INT; \
+	tmp_file=$$(mktemp); \
+	$(MAKE) konduit-cleanup; \
+	{ \
+			(tail -f -n0 "$$tmp_file" & ) | grep -q "postgres://"; \
+			postgres_url=$$(grep -o 'postgres://[^ ]*' "$$tmp_file"); \
+			echo "$$postgres_url"; \
+			sed -i '' -e "s|npq_registration_development|&\\n  url: \"$$postgres_url\"|g" config/database.yml; \
+	} & \
+	bin/konduit.sh -d s189p01-cpdnpq-pd-pg-mg -k s189p01-cpdnpq-pd-app-kv npq-registration-production-web -- psql > "$$tmp_file"
```

Is the `pd` here right?

```suggestion
	bin/konduit.sh -d s189p01-cpdnpq-mg-pg -k s189p01-cpdnpq-pd-app-kv npq-registration-production-web -- psql > "$$tmp_file"
```